packages/docusaurus-plugin-debug/src/index.ts

```diff
@@ -9,6 +9,8 @@ import type {LoadContext, Plugin} from '@docusaurus/types';
 import {docuHash, normalizeUrl, posixPath} from '@docusaurus/utils';
 import path from 'path';
 
+export const routeBasePath = '__docusaurus/debug';
```

Comment:
I didn't create plugin options for the sake of one parameter, so I decided to export related constant. Perhaps there is a better solution?

Reply:
Yeah, I think this is fine. I doubt if many know this plugin's existence anyways